docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -223,6 +223,7 @@ Query Parameters:
   -   `before`=(`<container id>` or `<container name>`)
   -   `since`=(`<container id>` or `<container name>`)
   -   `volume`=(`<volume name>` or `<mount point destination>`)
+  -   `network`=(`<network name>`)
```

Comment:
@sainath14 yes, I think searching by *name* would be the most common use-case, but as @vdemeester mentioned, people using scripts may actually use the ID. Also, I like it to be consistent.

Sorry for not catching this earlier!